### PR TITLE
add md5 feature output for biom

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by wasade** [comment]
Thanks! Can this logic be decomposed into a method in `redbiom.util`? Actually, do you think it would be better as an option to the underlying API method where if `hash_ids` is `True`, the IDs are hashed and the corresponding original ID is stored in the feature metadata? This way additional outputs don't need to be created, and the behavior is consistent whether interacting from the command line or Python API

**Comment by antgonza** [comment]
1. `Can this logic be decomposed into a method in redbiom.util?`: Sure
2. `Do you think it would be better as an option to the underlying API method where if hash_ids is True, the IDs are hashed and the corresponding original ID is stored in the feature metadata?`: To confirm, do you mean in the feature metadata in biom? If so, I rather have it as 2 outputs cause if in biom, AFAIK, there is no CLI way to recover those values in the feature metadata ...

File: redbiom/util.py [hunk 1]
```diff
@@ -368,3 +368,19 @@ def stems(stops, stemmer, string):
             yield stemmer.stem(word).lower()
         except Exception:
             continue
+
+
+def biom_md5_feature_ids(table, output):
```
Comment (wasade) [inline redbiom/util.py]: This method should not write a file as it's an internal call. As written, users of the Python API cannot use this method. Can ID mapping either be encoded within the BIOM table or written out within the CLI?

Also, would it be possible to add in a numpydoc string?

**Comment by wasade** [comment]
Thanks!!

**Pull request merged**